internal/kojiapi/api/openapi.yml

```diff
@@ -160,6 +192,10 @@ components:
         koji_import_logs: {}
         image_logs:
           type: array
+    Manifest:
```

Comment:
Will this definition still be valid after the DAG changes in osbuild, @gicmo? Maybe, we should just leave `Manifest` as a generic object here and add a note that it's a valid osbuild manifest with a link to https://www.osbuild.org/man/osbuild-manifest.5?

Reply:
Yeah, I'd leave it opaque.

Reply:
Yeah, I'd leave it opaque.

Reply:
This has already been addressed, but just for the record: might indeed change.